webpack.config.js

```diff
@@ -1,7 +1,13 @@
 const HtmlWebPackPlugin = require('html-webpack-plugin');
+const path = require('path');
 
 module.exports = {
   entry: './src/index.jsx',
+  output: {
+    path: path.resolve(__dirname, 'dist'),
+    filename: 'bundle.js',
```

Comment:
Good Work @alexxsanya , But Just asking do you have bundle.js file if yes could you show it ti me.Thanks

Reply:
> Good Work @alexxsanya , But Just asking do you have bundle.js file if yes could you show it time.Thanks

@jennizalwango kindly note that bundle.js is the build version of the application which generated by web pack & babel.